.github/workflows/acceptance_tests_broker.yaml

```diff
@@ -1,21 +1,23 @@
-name: Acceptance Tests
+name: Acceptance Tests (Broker)
 on:
   pull_request:
     types: [ opened, labeled, synchronize ]
 env:
   MAVEN_VERSION: 3.6.3
   MAVEN_SHA: c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
-  DEPLOYMENT_NAME: "app-autoscaler-${{ github.event.pull_request.number }}"
-  SERVICE_BROKER_NAME: "app-autoscaler-${{ github.event.pull_request.number }}servicebroker"
+  BUILDIN_MODE: false
+  SERVICE_OFFERING_ENABLED: true
+  check_name: "acceptance_test_check_broker"
```

Comment:
variable convention e.g CHECK_NAME

Reply:
this comes back from convention we're using lowe case for "vars local to script" ie. https://github.com/cloudfoundry/app-autoscaler-release/blob/main/ci/autoscaler/scripts/deploy-autoscaler.sh#L7